Resources/doc/cache-resolver/amazons3.rst

```diff
@@ -1,58 +1,103 @@
-AmazonS3Resolver
-================
 
-The AmazonS3Resolver requires the `aws-sdk-php`_ library. Open a command
-console, enter your project directory and execute the following command to
-download the latest stable version of this library:
+Amazon S3 Resolver
+==================
+
+.. _cache-resolver-amazon-s3:
+
+The ``AmazonS3Resolver`` resolver enables cache resolution using Amazon S3.
+
+Dependencies
+------------
+
+This cache resolver requires the `aws-sdk-php`_ library, which can be installed
+by executing the following command in your project directory:
 
 .. code-block:: bash
 
     $ composer require aws/aws-sdk-php
 
-This command requires you to have Composer installed globally, as explained
-in the `installation chapter`_ of the Composer documentation.
+.. note::
+
+    This command requires that `Composer`_ is installed globally, as explained in
+    their `installation documentation`_.
 
-Afterwards, you only need to configure some information regarding your AWS
-account and the bucket.
+Configuration
+-------------
+
+To begin, you must assign your Amazon key, secret, and bucket to their respective parameters.
 
 .. code-block:: yaml
 
+    # app/config/config.yml or app/config/parameters.yml
+
     parameters:
-        amazon_s3.key: 'your-aws-key'
-        amazon_s3.secret: 'your-aws-secret'
-        amazon_s3.bucket: 'your-bucket.example.com'
+        amazon_s3.key   : "your-aws-key"
```

Comment:
In Symfony Docs we align with white spaces instead of the `:`

```
amazon_s3.key   : "your-aws-key"  # <-- NO
amazon_s3.key :   "your-aws-key"  # <-- YES
```